### ISSUE TITLE
Project Licensing

### ISSUE THREAD

**Issue by ZXeno** [SteveSandersonMS/BlazorUnitTestingPrototype#11]
I don't see a license attributed to this repo. Are we free to utilize this code in another open-source project's tests with attribution/link back to this repo, or are there limitations and a preference to recommend against utilizing this code for testing?

**Comment by darcythomas** [comment]
Bumping this issue.

I would really like to play around with this, for some side project at work. Sadly for me work is really strict on not using something without a license. 

Legal obviously have their reasons!

Can we please have a license added so more people have a chance to play with this.

Thanks.

**Comment by egil** [comment]
Isnt the default for public repos here on github "open source", unless otherwise specified? I hope so, or at least I assume Steve has no problem with us taking this a building on it. 

I originally incorporated parts of this into https://github.com/egil/bunit, which does have a MIT license, so you can use that if you want, since I guess it'll be my problem, not yours, if I have broken any licensing agreements :)

**Comment by darcythomas** [comment]
Yeah technically it is open source. However it is unclear what the
conditions may be. Something can be open source under say the MIT or Apache
2.0 license, which we have lots of freedom to use. But then there is the
AGPL license, where we may end up obliged to publish our serverside secret
sauce code. (Legal feel a little twitchy about that too, for obvious
reasons).

It just makes things easier if there is a clear licence attached to the
code.

**Comment by egil** [comment]
@darcythomas I got a chance to ask Steve about this. Its fine to use, and there wont be any problems with the bits ive picked out for bUnit. So if you have any concerns with the prototype, you are fine to use bUnit as an alternative.
